lib/vrl/stdlib/benches/benches.rs

```diff
@@ -672,6 +673,35 @@ bench_function! {
     }
 }
 
+bench_function! {
+    is_empty => vrl_stdlib::IsEmpty;
+
+    empty_array {
+        args: func_args![value: value!([])],
+        want: Ok(true),
+    }
+
+    non_empty_array {
+        args: func_args![value: value!([1, 2, 3])],
+        want: Ok(true),
```

Comment:
false?

Reply:
Hmm, yeah, I missed that.

Reply:
Updated in [d55680c](https://github.com/vectordotdev/vector/pull/9732/commits/d55680c0e2ef0db35758bfbc334f2f283e8e3a1d)

Reply:
Isn't the assertion still in a mismatch here?